packages/storybook/stories/va-memorable-date-uswds.stories.jsx

```diff
@@ -0,0 +1,152 @@
+  // new Date as YYYY MM DD is giving the correct day selected
+  const dateInput = new Date(dateVal.split('-').join(' '));
+  function handleDateBlur() {
+    if (dateInput <= today) {
+      setErrorVal('Date must be in the future');
+    } else {
+      setErrorVal('');
+    }
+  }
+
+  return (
+    <>
+      <VaMemorableDate
+        uswds={uswds}
+        label={label}
+        name={name}
+        required={required}
+        error={errorVal}
+        value={dateVal}
+        onDateBlur={() => handleDateBlur()}
+        onDateChange={e => setDateVal(e.target.value)}
+      />
+      <hr />
+      <div>
+        This example has some custom validation logic to detect if the date
+        provided is in the future. The validation will occur when the component
+        is blurred ie: focus is removed from the component. This will cause the
+        error prop to be dynamically set if the parameters are not met.
+      </div>
+      <div className="vads-u-margin-top--2">
+        <a
+          href="https://github.com/department-of-veterans-affairs/component-library/tree/main/packages/storybook/stories"
+          target="_blank"
+        >
+          View validation code in our repo
+        </a>
+      </div>
+    </>
+  );
+};
+
+const I18nTemplate = ({ label, name, required, error, uswds, value }) => {
+  const [lang, setLang] = useState('en');
+
+  useEffect(() => {
+    document.querySelector('main').setAttribute('lang', lang);
+  }, [lang]);
+
+  return (
+    <div>
+      <button style={{fontSize: '16px'}} onClick={e => setLang('es')}>Español</button>
+      <button style={{fontSize: '16px'}} onClick={e => setLang('en')}>English</button>
+      <button style={{fontSize: '16px'}} onClick={e => setLang('tl')}>Tagalog</button>
+      <VaMemorableDate
+        uswds={uswds}
+        label={label}
+        name={name}
+        required={required}
+        error={error}
+        value={value}
+        onDateBlur={e => console.log(e, 'DATE BLUR FIRED')}
+        onDateChange={e => console.log(e, 'DATE CHANGE FIRED')}
+      />
+    </div>
+)};
+
+export const Default = Template.bind(null);
+Default.args = { ...defaultArgs };
+Default.argTypes = propStructure(memorableDateInputDocs);
+
+export const Error = Template.bind(null);
+Error.args = { 
+  ...defaultArgs, 
+  error: 'Error Message Example',
+};
+
+export const WithMonthSelect = Template.bind(null);
+WithMonthSelect.args = {
+  ...defaultArgs,
+  monthSelect: true,
+};
+
+export const ErrorWithMonthSelect = Template.bind(null);
+ErrorWithMonthSelect.args = {
+  ...defaultArgs,
+  error: 'Error Message Example',
+  monthSelect: true,
+};
+
+export const CustomValidation = CustomValidationTemplate.bind(null);
+CustomValidation.args = {
+  ...defaultArgs,
+  required: true,
+  value: '2022-04-19',
+};
+
+// export const Internationalization = I18nTemplate.bind(null);
+// Internationalization.args = {
+//   ...defaultArgs,
+//   error: 'Error Message Example',
+//   required: true,
+// };
```

Comment:
Commented out to disable i18n until bugs are fixed.
Tickets [1605](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1605) and [1488](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1488)